### DISCUSSION TITLE
make German achievments titles usable

### DISCUSSION THREAD

**Discussion by strikegun** [Lurkars/gloomhavensecretariat#359] [Ideas]
We are using the app a long time and we placed all won achievments in german in the app.
But now I see after some updates, only english spelled achievments are getting recognized by the app.
We have only the german game, we have here a little trouble

**Comment by Lurkars** [comment]
So you added German achievements manually? Because achievements should also get added automatically when finishing a scenario. Anyway you're right, that German  translation is missing. I will try to add it soon. If you are able to provide correct translation, that would help a lot. Besides I am native German speaker and can translate it myself, that may differ enough from original translation to not work out.

**Reply by strikegun** [reply]
I thought that you could be native german. ;-)
Yes sure I could support. I write you an email.
I think achievments was not from beginning automatic, or?

**Reply by Lurkars** [reply]
Yeah that's right. Automatic achievements were added a few month ago or so, but definitely not from start. I will add code (including migration) later today to support i18n achievements. The German translations are by me and not the official ones, so will still require some rework.
